app/js/components/tabs/tabcomponents/drugOrderComponent.jsx

```diff
@@ -118,11 +119,19 @@ class DrugOrderComponent extends Component {
                                 <span className="inline-label">From:</span>
                             </div>
                             <div className="col-sm-3">
-                                <input className="form-control" type="date" name="from-date" />
+                                <DatePicker
+                                    dateFormat="DD-MM-YYYY"
+                                    className="form-control"
+                                    name="from-date"
+                                />
                             </div>
                             <span className="inline-label">To:</span>
                             <div className="col-sm-3">
-                                <input className="form-control" name="to-date" type="date" />
+                                <DatePicker
+                                    dateFormat="DD-MM-YYYY"
```

Comment:
For now, let us go with that. But for the next version, we shall look into letting the end user change it.